gcovr/configuration.py

```diff
@@ -583,6 +583,17 @@ def choose(_cls, choices, default=None):
              "For text and HTML report.",
         action="store_true",
     ),
+    GcovrConfigOption(
+        "txt", ["-t", "--txt"],
```

Comment:
Single-letter options are a non-renewable resource. I'm not sure text output is important enough to get one. Opinions?

Reply:
I think you are right, `--html` also as no one letter option. I would vote for removing the `-t` here.

Reply:
I added it because of the `--xml` but I'm not a friend of short options. If I don't know the command a long option is easier to understand. I'll remove the short option.